std_msgs/Header header
string interface_name
bool success
string message
string[] joint_names
float64[] solution